### ISSUE TITLE
Error when running **attache deploy production --prune**

### ISSUE THREAD

**Issue by Rukudzo** [tpg/attache#17]
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to TPG\Attache\ReleaseService::__construct() must be an instance of TPG\Attache\Server, null given, called in /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/ReleasesPruneCommand.php on line 37 and defined in /Users/mac/.composer/vendor/thepublicgood/attache/src/ReleaseService.php:28
Stack trace:
#0 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/ReleasesPruneCommand.php(37): TPG\Attache\ReleaseService->__construct(NULL)
#1 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/Command.php(111): TPG\Attache\Console\ReleasesPruneCommand->fire()
#2 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/DeployCommand.php(67): TPG\Attache\Console\Command->execute(Object(Symfony\Component\Console\Input\ArrayInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/Command.php(111): TPG\Attache\Console\DeployCommand->fire()
#4 /Users/mac/.composer/vendor/symfony/cons in /Users/mac/.composer/vendor/thepublicgood/attache/src/ReleaseService.php on line 28

Fatal error: Uncaught TypeError: Argument 1 passed to TPG\Attache\ReleaseService::__construct() must be an instance of TPG\Attache\Server, null given, called in /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/ReleasesPruneCommand.php on line 37 and defined in /Users/mac/.composer/vendor/thepublicgood/attache/src/ReleaseService.php:28
Stack trace:
#0 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/ReleasesPruneCommand.php(37): TPG\Attache\ReleaseService->__construct(NULL)
#1 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/Command.php(111): TPG\Attache\Console\ReleasesPruneCommand->fire()
#2 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/DeployCommand.php(67): TPG\Attache\Console\Command->execute(Object(Symfony\Component\Console\Input\ArrayInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Users/mac/.composer/vendor/thepublicgood/attache/src/Console/Command.php(111): TPG\Attache\Console\DeployCommand->fire()
#4 /Users/mac/.composer/vendor/symfony/cons in /Users/mac/.composer/vendor/thepublicgood/attache/src/ReleaseService.php on line 28

Issue seems to be with the **--prune** section of the command
works if you run **attache deploy production** then **attache releases:prune production**

**Comment by warrickbayman** [comment]
Thanks.

Seems to be an issue in the `DeployCommand` here:

```php
        if ($this->option('prune')) {
            $command = new ReleasesPruneCommand(null, $this->config);
            $command->execute(new ArrayInput([
                'server' => 'production',
                '--force' => true,
            ]), $this->output);
        }
```

The `production` string shouldn't be hard coded. For now, use the `releases:prune` command.

**Comment by warrickbayman** [comment]
Okay, so the problem is actually the call to the `execute` method. It should be:

```php
$command->run(new ArrayInput([
    'server' => 'production',
    '--force' => true,
 ]), $this->output);
```
Will be fixed 0.4.4.
